packages/jupyter-ai/jupyter_ai/completions/handlers/llm_mixin.py

```diff
@@ -26,8 +26,8 @@ def __init__(self, *args, **kwargs) -> None:
         self._llm_params = None
 
     def get_llm(self) -> Optional[BaseProvider]:
-        lm_provider = self.jai_config_manager.lm_provider
-        lm_provider_params = self.jai_config_manager.lm_provider_params
+        lm_provider = self.config_manager.completions_lm_provider
+        lm_provider_params = self.config_manager.completions_lm_provider_params
```

Comment:
It's a bit weird that the error message says `'ConfigManager' object ...` instead of `None` however. Maybe Jupyter Server also sets `self.config_manager`?

Reply:
Yes, sorry - a typo when resolving conflicts last time. It is fixed with 5ac3cf307dad46c409c5f4ea3673ceefe9af4a05.